miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/form/QcTableTag.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2002-2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.bbsrc.tgac.miso.webapp.util.form;
+
+import java.util.stream.Collectors;
+
+import org.springframework.web.servlet.tags.RequestContextAwareTag;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import uk.ac.bbsrc.tgac.miso.core.data.QualityControllable;
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+
+@SuppressWarnings("serial")
+public class QcTableTag extends RequestContextAwareTag {
+  private static long uniqueId;
+  private Object item;
+
+  @Override
+  protected int doStartTagInternal() throws Exception {
+
+    QualityControllable<?> qcItem = (QualityControllable<?>) item;
+    ObjectMapper mapper = new ObjectMapper();
+
+    pageContext.getOut().append(String.format(
+        "<br/><h1>QCs</h1><table id='qc%1$d' class='display no-border ui-widget-content'></table><script type='text/javascript'>jQuery(document).ready(function () { ListUtils.createStaticTable('qc%1$d', ListTarget.qc('%2$s'), { ownerId : %3$d }, %4$s);});</script>",
+        uniqueId++, qcItem.getQcTarget(), qcItem.getId(),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)